.gitignore

```diff
@@ -21,3 +21,5 @@ Thumbs.db
 .idea
 pubspec.lock
 .vscode/
+docs/doxygen/
```

Comment:
I'd prefer the docs went inside the `out` directory instead. That is already in the ignores and is usually the place we put generated build artifacts.

Reply:
Doxygen isn't smart enough to create directories so I could only put them in ../out/html, not ../out/<some-directory>/html.  That doesn't seem ideal.

Reply:
Sounds good.